### DISCUSSION TITLE
70Mb apk file size

### DISCUSSION THREAD

**Discussion by ZincTelluride** [PaulWoitaschek/Voice#1503] [Q&A] [answered]
Why is the file size of voice 70mb on the f-droid store and 14mb in the google playstore?
Would you be so kind as to explain it to me

**Comment by PaulWoitaschek** [comment, marked as answer]
Because fdroid does not Support App bundles:.
https://forum.f-droid.org/t/aab-android-bundles/6098
https://developer.android.com/guide/app-bundle/
